Modules/Wiki/classes/class.ilObjWiki.php

```diff
@@ -769,13 +769,13 @@ public function createWikiPage(
         return $page;
     }
 
-    public static function getAdvMDSubItemTitle($a_obj_id, $a_sub_type, $a_sub_id)
+    public static function getAdvMDSubItemTitle($a_obj_id, $a_sub_type, $a_sub_id) : string // TODO PHP8-REVIEW Type hints are missing here
```

Comment:
Type hints are missing here

Reply:
They are missing in the metadata service.